ansible/roles/ocp-workload-istio-prometheus-demo/tasks/workload.yml

```diff
@@ -101,27 +125,34 @@
 
 - name: Create Hello World Application Project
   shell: |
-    oc new-project {{ocp_project_name}}-prod --display-name='{{ocp_project_display_name}} Production'
+    oc new-project {{ ocp_project_name }}-prod --display-name='{{ ocp_project_display_name }} Production'
     oc delete limitrange --all
 
 - name: Setup Service Account Access
-  shell: |
-    oc adm policy add-scc-to-user anyuid -z default -n {{ocp_project_name}}-prod
-    oc adm policy add-scc-to-user privileged -z default -n {{ocp_project_name}}-prod
-    oc policy add-role-to-user edit system:serviceaccount:{{ocp_project_name}}-cicd:jenkins -n {{ocp_project_name}}-prod
+  shell: >-
```

Comment:
In this and the next two shell commands, there are multiple ways of stringing commands together. Please pick one and maintain consistency.